### ISSUE TITLE
& not properly escaped in featureXML

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#2608]
& is not properly escaped in featureXML
10:54:34 NOTICE: TextExporter of node #10 started. Processing ...
..\..\..\OpenMS\src\openms\source\FORMAT\HANDLERS\XMLHandler.cpp(103): While loading 'X.featureXML': unterminated entity reference 'coxG:'( in line 3397 column 70)

```
			<ProteinHit id="PH_670" accession="ABV95450.1|Dshi_3719|coxL&coxG:" score="0.943420648574829" sequence="">
				<UserParam type="string" name="isDecoy" value="false"/>
				<UserParam type="string" name="target_decoy" value="target"/>
				<UserParam type="float" name="_score" value="0"/>
			</ProteinHit>
```

**Comment by hroest** [comment]
related to #24 , try to wrap it in `writeXMLEscape()`

**Comment by timosachsenberg** [comment]
fixed in  #2626
